### DISCUSSION TITLE
If I want to add a dataset to the STAC-API where should I submit an issue?

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#766] [Data] [answered]
Hey @wildintellect,

Should we still submit issues to https://github.com/MAAP-Project/ZenHub/ if we want to add a new dataset to the STAC?  Not sure now that we have discussions.

**Comment by wildintellect** [comment, marked as answer]
@mccabete yes https://github.com/MAAP-Project/Community/issues (Zenhub is now Community, old url will redirect), more specifically choose the Data Request form on https://github.com/MAAP-Project/Community/issues/new/choose.

We are working on [updating](https://github.com/MAAP-Project/Community/pull/712) this form to include more STAC specific questions but starting a ticket there as is will suffice.
